src/content/guidelines/color/palettes.mdx

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
this looks great to review in context!

lets make sure to remove before merging

Reply:
Do you want me to remove the intermediary component in main src folder too? It parses the string into an array, which is required in the addons component

Reply:
Oh yea, we should be able to just import from addons no?

Reply:
Mdx doesn't seem to allow passing an array, unless it's set up in frontmatter using the templates/page.js. I didn't want to mess with that. If you have a better way of doing it, let me know.

Reply:
Oh ok, maybe add a readme file for this component then that explains how to use?

Reply:
I have removed the palettes.mdx file and the SwatchPaletteWIdget from the src files and added a read me with full instructions in the addons-website components folder